src/Microsoft.SqlTools.ServiceLayer/Utility/CommandOptions.cs

```diff
@@ -102,33 +104,44 @@ public string Usage
                     "Microsoft.SqlTools.ServiceLayer.exe " + Environment.NewLine +
                     "   Options:" + Environment.NewLine +
                     "        [--enable-logging]" + Environment.NewLine +
+                    "        [--log-dir **] (default: current directory)" + Environment.NewLine +
                     "        [--help]" + Environment.NewLine +
                     "        [--locale **] (default: 'en')" + Environment.NewLine,
                     ErrorMessage);
                 return str;
             }
         }
 
-        private void SetLocale(string locale)
+        private void SetLoggingDirectory(string loggingDirectory)
         {
             try
             {
-                // Creating cultureInfo from our given locale
-                CultureInfo language = new CultureInfo(locale);
-                Locale = locale;
-
-                // Setting our language globally 
-                CultureInfo.CurrentCulture = language;
-                CultureInfo.CurrentUICulture = language;
+                if (string.IsNullOrWhiteSpace(loggingDirectory))
+                {
+                    return;
+                }
 
-                // Setting our internal SR culture to our global culture
-                SR.Culture = CultureInfo.CurrentCulture;
+                this.LoggingDirectory = Path.GetFullPath(loggingDirectory);
             }
-            catch(Exception ex)
+            catch (Exception ex)
             {
-                // Warn user of invalid locale, and fall back to english 
+                // Warn user of invalid logging directory, and fall back to the default
                 Console.WriteLine(ex);
```

Comment:
>Console.WriteLine(ex); [](start = 16, length = 22)

I'm thinking I should remove this and log a warning instead. By writing to stdout like this, we're going to confuse any client expecting json-rpc.